src/js/components/sharedComponents/DatePicker.jsx

```diff
@@ -227,6 +227,26 @@ export default class DatePicker extends React.Component {
                 before: this.props.opposite.toDate()
             });
         }
+        else if (this.props.type === 'startDateBulk' && this.props.opposite) {
```

Comment:
If `DatePicker` is a such a commonly used element across the site, does it make sense to hard-code a bulk download-specific ruleset into the shared component? Maybe we should modify the component so it can accept validation rules from parent components (via props) and put the bulk download rules in the bulk download code?

Reply:
Yep. I moved the logic of `disabledDays` from here to the parents that implement `DatePicker`.